src/js/index.js

```diff
@@ -0,0 +1 @@
+
```

Comment:
Not sure if this will break the current file that is in main, maybe add the code from main in here just to be on the safe side+?